### PR TITLE
Added description how to rotate ssh keys

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added documentation how to rotate ssh keys for kubernetes cluster

## Why
* Share knowledge

### CODE REVIEW

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- faq
+---
+
+# Question
+
+How to rotate ssh keys for Kubernetes?
+
+# Answer
+
+Geodesic creates ssh keys with [terraform-aws-key-pair]({{< relref "terraform-modules/security/terraform-aws-key-pair.md" >}}) module.
```
Comment (aknysh) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: Better to say:
We provision SSH keys in `geodesic` using [terraform-aws-key-pair]({{< relref "terraform-modules/security/terraform-aws-key-pair.md" >}}) module

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Following this [documentation](https://www.terraform.io/docs/providers/tls/r/private_key.html#generating-a-new-key) you need to taint ssh key with terraform, then provision new one and update the cluster.
+
+To rotate ssh keys please follow this instruction:
```
Comment (aknysh) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: To rotate SSH keys, follow these instructions:

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+{{% include-code-block title="Run the Geodesic Shell" file="geodesic/module/examples/start-geodesic-shell.txt" %}}
+
+Then login to AWS by running `assume-role`:
+
+{{% include-code-block title="Assume role" file="geodesic/module/examples/assume-role.txt" %}}
+
+## Configure `kubectl` and `helm`
+
+When you start the Geodesic shell, you will need to export the `kubecfg` which provides the TLS client certificates necessary for `kubectl` and `helm` to authenticate with the cluster.
```
Comment (aknysh) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: Comma before `which`:

`kubecfg`, which provides

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+{{% dialog type="code-block" icon="fa fa-code" title="Export kops config" %}}
+```
+✅   (example-staging-admin) ~ ➤  kops export kubecfg
+kops has set your kubectl context to us-west-2.staging.example.com
+```
+{{% /dialog %}}
+
+(Note, in older versions of `kops` you will need to pass the cluster name, so run `kops export kubecfg $KOPS_CLUSTER_NAME`)
+
+## Recreate `ssh keys`
```
Comment (aknysh) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: Recreate SSH key

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+```shell
+cd /conf/kops
+init-terraform
+terraform taint --module ssh_key_pair tls_private_key.default
+terraform apply
+```
+
+
+## Update Kubernetes cluster
```
Comment (aknysh) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: Capitalize:
Update Kubernetes Cluster

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+Following this [instrunctions](https://github.com/kubernetes/kops/blob/master/docs/security.md)
```
Comment (aknysh) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: Following these

Comment (aknysh) [reply content/faq/how-rotate-kubernetes-ssh-keys.md]: Better to reword it:

According to the [instrunctions](https://github.com/kubernetes/kops/blob/master/docs/security.md), execute the following commands to apply the new SSH key to the Kubernetes cluster:

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
+-=----
```
[the remark this answers is not in the document]
Comment (osterman) [reply content/faq/how-rotate-kubernetes-ssh-keys.md]: Rename file to `how-to-rotate-kops-ssh-keys.md`

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+title: "How to rotate ssh keys for Kubernetes?"
```
Comment (osterman) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: I think it should be: 

> How to rotate SSH keys for a Kops cluster?

@aknysh ?

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+description: "Learn how to rotate ssh keys for kubernetes."
```
Comment (osterman) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: "Learn how to rotate ssh keys on a kops cluster by generating new ones and then performing a rolling-update on the cluster to apply the changes."

File: content/faq/how-rotate-kubernetes-ssh-keys.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+tags:
+- kops
+- geodesic
+- kubernetes
+- faq
+---
+
+# Question
+
+How to rotate ssh keys for Kubernetes?
```
Comment (osterman) [inline content/faq/how-rotate-kubernetes-ssh-keys.md]: How do we rotate the master SSH keys on a Kubernetes cluster provisioned with `kops`?

**Pull request merged**